exercises/two-fer/test/two_fer_tests.erl

```diff
@@ -0,0 +1,15 @@
+-module(two_fer_tests).
+
+-define(TESTED_MODULE, (sut(two_fer))).
+-define(TEST_VERSION, 1).
+-include("exercism.hrl").
+
+
+say_you_test() ->
+  ?assertEqual("One for you, one for me.", ?TESTED_MODULE:two_fer("")).
```

Comment:
As stated above, I do personally treat the empty string as a name given, and like to have this test as `?TESTED_MODULE:two_fer()`.

We have the ability to create multiple functions with the same name but different arities, so we should use it.